craft/pwm/src/PWM.scala

```diff
@@ -74,11 +74,6 @@ object NPWMTop {
     val PWM = NPWMTopBase.attach(c)(bap)
 
     // User code here
-    implicit val p: Parameters = bap.p
-
-      // route PWM signals to the testharness
-      val PWMNode = BundleBridgeSink[NPWMTopIO]()
-      PWMNode := PWM.ioBridgeSource
```

Comment:
each `BundleBridgeSource` can only be connected to exactly one `BundleBridgeSink` so I removed this since the `NcapctlTop.attach` method already creates a `BundleBridgeSink` for the `PWM`